deployment/legacy-migration/maintenance-worker/migration/mongo/export.sh

```diff
@@ -24,9 +31,12 @@ ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${
 export_collection "badges"
 export_collection "categories"
 export_collection "comments"
-export_collection "contributions"
+export_collection_query "contributions" "{'type': 'DELETED'}" "DELETED"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T20:52:12Z" title="Friday, June 21st 2019, 10:52:12 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,9 +31,12 @@ ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${
 export_collection "badges"
 export_collection "categories"
 export_collection "comments"
-export_collection "contributions"
+export_collection_query "contributions" "{'type': 'DELETED'}" "DELETED"
```


Apparently, the `type` got overwritten when sb. deleted a post or cando and we cannot distinguish it anymore. I would simply not import `DELETED` data at all.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-06-24T10:15:15Z" title="Monday, June 24th 2019, 12:15:15 pm +02:00">Jun 24, 2019</time>_

---


yes, i export them only - i do not import the deleted data.